action/rewards/withdraw.go

```diff
@@ -0,0 +1,133 @@
+package rewards
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"github.com/tendermint/tendermint/libs/kv"
+
+	"github.com/Oneledger/protocol/action"
+	"github.com/Oneledger/protocol/action/helpers"
+	"github.com/Oneledger/protocol/data/balance"
+	"github.com/Oneledger/protocol/data/rewards"
+)
+
+type Withdraw struct {
+	ValidatorAddress        action.Address `json:"validatorAddress"`
+	ValidatorSigningAddress action.Address `json:"validatorSigningAddress"`
```

Comment:
If we use a separate signing address, how can we check if it is owned by the validator?

Reply:
Yeah , the assumption was ,if someone else wants to pay for a trasfer of money to the validator ,he can ,the funds still go to the validator address ,but you are right , I think ill do this through olclient  ,That would make it more secure